src/GraphQL.DataLoader/GraphQL.DataLoader.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <Description>GraphQL data loader implementation</Description>
```

Comment:
`data loader` or `dataloader` ? 🤔

Reply:
Whatever you think. 🤷‍♂️

Reply:
https://github.com/graphql/dataloader

search `dataloader` -> 83 matches
search `data loader` -> 0 mathes

😄

Reply:
ok so dataloader then?

Reply:
```suggestion
    <Description>GraphQL dataloader implementation</Description>
```

Reply:
Yes, everywhere.

Reply:
Well, even `DataLoader`. It is in such a register that this name is used.

Reply:
```suggestion
    <Description>GraphQL DataLoader implementation</Description>
```